docs/src/gcode/machining-center_fr.adoc

```diff
@@ -77,16 +77,12 @@ l'opérateur corriger les vitesses nécessaires pour la broche et
 l'avance travail, il peut ainsi augmenter ou réduire les vitesses
 programmées.
 
-=== Bouton d'effacement de bloc[[sec:Bouton-effacement-de-block]]
-
-(((Bouton effacement de bloc)))
+=== Bouton d'effacement de bloc
 
 Une machine à commande numérique peut être équipée d'un bouton d'effacement de
 bloc. Voir la section <<sec:Interaction-effacement-de-bloc,effacement de bloc>>.
 
-=== Bouton d'arrêt optionnel du programme[[sec:arret-optionnel]]
-
-(((arrêt optionnel)))
+=== Bouton d'arrêt optionnel du programme
```

Comment:
```suggestion
=== Bouton d'arrêt optionnel du programme [[sec:arret-optionnel]](((Arrêt Optionnel)))
```

Reply:
If the original did not have such markers then I removed them from the translation.